lib/msal-core/src/cache/AuthCache.ts

```diff
@@ -206,6 +208,24 @@ export class AuthCache extends BrowserStorage {// Singleton
         return results;
     }
 
+    /**
+     * Get idToken from the cache
+     */
+    getIdToken(clientId: string, homeAccountIdentifier: string, authority: string): AccessTokenCacheItem {
+        const idTokenKey: AccessTokenKey = {
+            authority: authority,
+            clientId: clientId,
+            scopes: undefined,
+            homeAccountIdentifier: homeAccountIdentifier
+        };
+
+        const idTokenKeyString = JSON.stringify(idTokenKey);
+        const idToken = this.getItem(idTokenKeyString);
+        const idTokenValue = JSON.parse(idToken) as AccessTokenValue;
+        
+        return new AccessTokenCacheItem(idTokenKey, idTokenValue);
```

Comment:
Can we add a comment here explaining why a function for getting an idToken is returning an object called AccessToken?

Reply:
What's your opinion on renaming AccessTokenCacheItem to TokenCacheItem? Other than the overhead involved, is there a philosophical/technical reason not to make it a more general type?